get_github_coderabbit_review/tests/performance/test_performance.py

```
@@ -0,0 +1,557 @@
+        """Set up test fixtures."""
+        self.temp_dir = tempfile.mkdtemp(prefix="perf_test_")
+        self.process = psutil.Process(os.getpid())
+        self.initial_memory = self.process.memory_info().rss
+    
+    def tearDown(self):
+        """Clean up test fixtures."""
+        if os.path.exists(self.temp_dir):
+            for file in os.listdir(self.temp_dir):
+                try:
+                    os.unlink(os.path.join(self.temp_dir, file))
+                except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)